### ISSUE TITLE
Migrar HubAPI para DigitalOcean

### ISSUE THREAD

**Issue by caio-emidio** [aHub-Tech/hub-api#18]
Apos criar dockerFile. Migrar arquivos e garantir que funciona. 

Um adendo eh verificar se o MongoDB tem acesso para testes via ROBO 3T.

**Comment by DevRadhy** [comment]
Issues a serem resolvidas antes #19 e #17 

A Issue #17 depende das Issues #21 e #20

**Comment by caio-emidio** [comment]
Estarei off pelas proximas 3 semanas. Tentem ver com a @Isabelar07 . Caso ela nao consiga me avisem q eu tento dar prioridade pra isso no meio das minhas ferias.

Volto a dizer q no meu ponto de vista. Migrar o backend da hub eh prioridade para o nosso calendario.

**Comment by vcwild** [comment]
> Issues a serem resolvidas antes #19 e #17
> 
> A Issue #17 depende das Issues #21 e #20

Concordo com as ressalvas, porém acredito que as issues #20 e #21 tratam do mesmo assunto

**Comment by DevRadhy** [comment]
> > Issues a serem resolvidas antes #19 e #17
> > A Issue #17 depende das Issues #21 e #20
> 
> Concordo com as ressalvas, porém acredito que as issues #20 e #21 tratam do mesmo assunto

A Issue #21 apresenta o problema e a Issue #20 tanta resolver, mas em um contexto geral pertencem ao mesmo assunto

**Comment by DevRadhy** [comment]
**Issues #20** e **#21** resolvidas pela @Isabelar07, no **PR #22**
